src/app/pages/rpc-send-transfer/use-rpc-send-transfer.ts

```diff
@@ -13,17 +14,18 @@ export function useRpcSendTransferRequestParams() {
   return useMemo(
     () => ({
       ...defaultParams,
-      address: initialSearchParams.get('address') ?? '',
-      amount: initialSearchParams.get('amount') ?? '',
       requestId: initialSearchParams.get('requestId') ?? '',
+      recipients: JSON.parse(
+        initialSearchParams.get('recipients') ?? '[]'
+      ) as RpcSendTransferRecipient[],
```

Comment:
Dangerous operation, can throw.

Query Params have their own way of handling arrays, by repeating the identifer